api/graphql/mutations.js

```diff
@@ -26,6 +26,16 @@ function convertGraphqlCreateData (data) {
   }, data))
 }
 
+function convertGraphqlUserSettingsData (data) {
```

Comment:
might be worthwhile to convert this one to behaving as a promise to match the one above, or converting the one above. feels awkward to have the two behave opposingly